tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Ricci.cpp

```diff
@@ -38,22 +41,32 @@ void test_compute_item_in_databox(const DataType& used_for_size) {
   const auto d_christoffel_2nd_kind = make_with_random_values<
       tnsr::aBcc<DataType, Dim, Frame::Inertial, TypeOfIndex>>(
       nn_generator, nn_distribution, used_for_size);
+  const auto inverse_metric = make_with_random_values<
```

Comment:
We have some functions that make "sensible" spatial metrics. I'd use those, and then invert it. The current code doesn't require the spatial metric to be reasonable, but someone might be in for a nasty surprised if they try to use this and it has the wrong signed determinant, etc.